app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -103,6 +103,11 @@
         BackKeyHandleable, ScreenshotObserver.OnScreenshotListener {
 
     public static final String FRAGMENT_TAG = "browser";
+
+    /** Custom data that is passed when calling {@link TabsSession#addTab(String, Bundle)} */
+    public static final String EXTRA_NEW_TAB_SRC = "extra_bkg_tab_src";
+    public static final int SRC_CONTEXT_MENU = 0;
```

Comment:
I don't have strong opinion: It also make sense to me if these two lines were placed in ContextMenu

Reply:
The reason I put it here is because 
1. BrowserFragment is the last receiver of the event, and is also the only one that knows how to react to this field. 
2. The number of sources may increase in the future(e.g. we might want to change UI when  a tab is added by javascript). Putting all these src definitions at one place can make maintenance easier.